src/SystemsRx/Executor/Handlers/Conventional/ReactToEventSystemHandler.cs

```diff
@@ -23,7 +23,7 @@ public ReactToEventSystemHandler(IEventSystem eventSystem)
         {
             EventSystem = eventSystem;
             _systemSubscriptions = new Dictionary<ISystem, IDisposable>();
-            _setupSystemGenericMethodInfo = typeof(ReactToEventSystemHandler).GetMethod(nameof(SetupSystemGeneric), BindingFlags.Instance | BindingFlags.NonPublic);
+            _setupSystemGenericMethodInfo = typeof(ReactToEventSystemHandler).GetMethod(nameof(SetupSystemGeneric));
```

Comment:
Does this still work as expected without the explicit binding flags?

Reply:
AH right I see you made it public and then just get the method, and the tests confirm this works. Thanks for PR will accept it now.